src/main/java/bsise/server/report/ReportController.java

```diff
@@ -0,0 +1,32 @@
+package bsise.server.report;
+
+import bsise.server.report.daily.DailyReportDto;
+import bsise.server.report.daily.DailyReportResponseDto;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+
+@RestController
+@RequestMapping(path = "/api/v1/reports", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
```suggestion
@RequestMapping(path = "/api/v1/reports")
```

`@RestController` 이게 있으니까 이렇게 수정 부탁드려도 될까요?